src/assets.rs

```diff
@@ -0,0 +1,102 @@
+use crate::{key_value_store::*, persistence::EncryptingResourceAdapter};
```

Comment:
Almost forgot, this file should have Rust docs for the module and all public structs, fields, and methods